src/components/Composer/index.js

```diff
@@ -328,16 +323,21 @@ class Composer extends React.Component {
      * divide by line height to get the total number of rows for the textarea.
      */
     updateNumberOfLines() {
-        const computedStyle = window.getComputedStyle(this.textInput);
-        const lineHeight = parseInt(computedStyle.lineHeight, 10) || 20;
-        const paddingTopAndBottom = parseInt(computedStyle.paddingBottom, 10)
-            + parseInt(computedStyle.paddingTop, 10);
+        // Hide the composer expand button so we can get an accurate reading of
+        // the height of the text input
+        this.props.setIsFullComposerAvailable(false);
 
         // We have to reset the rows back to the minimum before updating so that the scrollHeight is not
         // affected by the previous row setting. If we don't, rows will be added but not removed on backspace/delete.
         this.setState({numberOfLines: 1}, () => {
```

Comment:
Just to make sure I understand, we're setting the number of lines to 1, and then once that's done we're updating the number of lines again to be correct?

Reply:
Yeah. For some reason we have to set the number of lines to 1 to properly compute the scrollHeight, and I found that I also had to remove the expand / collapse button.